### PR TITLE
Add a worth-prototyping position for FedCM

### LINKED ISSUE #618: FedCM (was WebID)
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: [Federated Credential Management API](https://github.com/fedidcg/FedCM)
* Specification or proposal URL: 
    * [explainer](https://github.com/fedidcg/FedCM/blob/main/explainer.md)
    * [specification](https://fedidcg.github.io/FedCM/)
    * [TPAC2021](https://watch.videodelivery.net/29bab61e04e8cabf1517e5885c9fe4cf) presentation (good introduction)
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional): 
    * Since we started, we've discussed this proposal with a few mozillians: 
    * stpeter (general point of contact)
    * @martinthomson (overall guidance)
    * tvyas, johannhof (early privacy discussions)
    * danmills, rfkelly, arthur (on mozilla personas)

### Other information

Various other reviews:

* [WICG thread](https://discourse.wicg.io/t/proposal-webid-privacy-preserving-federated-sign-in-api/4661)
* [Federation and Browsers Workshop](https://github.com/fedidcg/FedCM/blob/main/meetings/2021/25-26_May_2021.md)
* [Intent To Prototype](https://groups.google.com/a/chromium.org/g/blink-dev/c/2B4TJ7j2U4M/m/1X5T3OszCAAJ)
* [Intent To Experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/kws-gltC5us)
* [FedID CG](https://www.w3.org/community/fed-id/)
* early [TAG review](https://github.com/w3ctag/design-reviews/issues/622)
* [Ready For Trial](https://groups.google.com/a/chromium.org/g/blink-dev/c/jlV_1m7uUAg)
* spec [TAG review](https://github.com/w3ctag/design-reviews/issues/718)
* [WebKIt's Position Thread](https://lists.webkit.org/pipermail/webkit-dev/2022-March/032153.html)

### PR DESCRIPTION
**Pull request by martinthomson**
There is a lot to unpack here.  We can't say that this is an unqualified
good, particularly given the effect that it might have on IdPs and RPs.
But we can at least signal some amount of positive interest.

cc @bvandersloot-mozilla

Closes #618.

### CODE REVIEW

File: activities.json [hunk 1]
```diff
@@ -472,6 +472,19 @@
     "title": "Event Timing API",
     "url": "https://wicg.github.io/event-timing/"
   },
+  {
+    "ciuName": null,
+    "description": "A Web Platform API that allows users to login to websites with their federated accounts in a privacy preserving manner.",
+    "id": "fedcm",
+    "mdnUrl": null,
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1782066",
+    "mozPosition": "worth prototyping",
```
Comment (martinthomson) [inline activities.json:481]:
```suggestion
    "mozPosition": "positive",
```

File: activities.json [hunk 2]
```diff
@@ -472,6 +472,19 @@
...
...Management API puts the browser in control of managing cross-site logins.  Browsers can use this API as a way to give web users better ability to control and monitor how their identity - and any information related to their identity - is exchanged between sites.  Including the browser in a mediating role will adversely affect some cross-site interactions, in some cases making them less efficient or even less usable.  However, Mozilla considers it imperative that this change occur so that users can be granted control - and awareness - of all instances where their information is transferred between sites.  This proposal provides browsers with the opportunity to provide these capabilities.  Note that Mozilla also wants to acknowledge an important privacy compromise in the proposal: identity providers learn when and where the identity they provide is used.  Though alternative designs might be technically possible, this approach recognizes the security benefits gained by allowing identity providers the ability to audit logins.  Furthermore, though this design enables an authorized identity to track cross-site activity, it only does so with the direct permission and knowledge of users.",
```
Comment (bvandersloot-mozilla) [inline activities.json:482]: This looks good. It captures the core conflicts in the space and how they are resolved by this proposal at a high level.

**Review by bvandersloot-mozilla — APPROVED** [review]
LGTM

**Review by tantek — APPROVED** [review]
LGTM and accurately summarizes concerns noted in issue #618.

**Comment by martinthomson** [comment]
Thanks Tantek.  Moving on.

**Pull request merged**